Sender/BMS_Streaming.Test/BatteryDataStreamingTest.cs

```diff
@@ -0,0 +1,41 @@
+using Newtonsoft.Json.Linq;
+using System;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace BMS_Streaming.Test
+{
+    public class BatteryDataStreamingTest
```

Comment:
Focus on testing your functionality, instead of just passing tests - the most crucial is the output format.
In the absence of a test for the output format, any future mistakes will be caught only while integrating with the receiver.